tests/api/metrics/test_job.py

```diff
@@ -47,6 +47,7 @@ def test_multiple_job_runs(client, workflow_job, headers):
 
     metrics = client.get("/metrics")
     assert metrics.status_code == 200
+    assert workflow_job["workflow_job"]["workflow_name"] in metrics.text
```

Comment:
This works, but might be risky if something else is named `test` in the workflow_job.

We should be able to find `workflow_name=test` in the metrics
```suggestion
    assert f"workflow_name={workflow_job['workflow_job']['workflow_name']" in metrics.text
```